app/jjit.py

```diff
@@ -0,0 +1,135 @@
+from bs4 import BeautifulSoup
+from selenium import webdriver
+from selenium.webdriver.chrome.service import Service
+
+import pandas as pd
+import time
+import re
+import os
+
+
+def scrap_data_jjit(driver, offers, links):
```

Comment:
Teraz jak doczytałem całą logikę programu, to wychodzi na to, że ta funkcja *parsuje* HTML a nie *scrapuje*. Subtelna, ale istotna różnica. Scrapowanie to cały proces pobierania danych, przede wszystkim requestowanie.